src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -56,9 +56,14 @@ function ResultsView(props: ResultsViewProps) {
     const urlSearchParams = new URLSearchParams(location.search);
     const repos = urlSearchParams.get('repos')?.split(',');
     const revs = urlSearchParams.get('revs')?.split(',');
+    const framework = urlSearchParams.get('framework');
 
     if (revs && repos) {
-      void dispatchFetchCompareResults(repos as Repository['name'][], revs);
+      void dispatchFetchCompareResults(
+        repos as Repository['name'][],
+        revs,
+        framework as string,
```

Comment:
Isn't it a string already? It's really not clear to me what we pass as the framework value...

Reply:
It's the framework id from the URL param

Reply:
It's the value from the URL param. If the param is not mentioned in URL then it could be null.